docs/versioning/versioning-overview.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+nav_order: 2
+title: Versioning Overview 
+has_children: false
+parent: Versioning
+---
+# Versioning Overview 
+
+Provena provides capability for versioning of a relevant subset of entity types in the Registry, including datasets. The purpose of this feature is to allow users to create and manage versions of registered items in Provena. Users can version items to mark items at a point in their development or release lifecycle. Users can then manage and refer to versions of an item. This can then provide a more accurate provenance trail, e.g. version 1 of Dataset X uses a different configuration of a modelling software suite to version 2 of Dataset X.
+
+Other use cases include:
+* Marking a previous version of an item as deprecated or erroneous
```

Comment:
Can users actually do this, though? At least, does Provena support it?

Reply:
mm, not yet i think. however, this was some of the original use cases as a first class feature. however, users can still edit the description to say this. maybe we could edit this statement to match.

Reply:
edited the statement now